src/xma/src/xmaplugin/xmaplugin.cpp

```diff
@@ -412,20 +302,19 @@ xma_plg_buffer_read(XmaSession s_handle,
         return XMA_SUCCESS;
     }
 
-    rc = xclSyncBO(b_obj_priv->dev_handle, b_obj_priv->boHandle, XCL_BO_SYNC_BO_FROM_DEVICE,
-                   size, offset);
-    if (rc != 0)
-    {
-        xma_logmsg(XMA_ERROR_LOG, XMAPLUGIN_MOD, "xma_plg_buffer_read failed. dev_index: %d. xclSyncBO failed. Check device logs for more info. Error: %d", b_obj_priv->dev_index, rc);
-        return XMA_ERROR;
+    try {
+        b_obj_priv->xrt_bo.sync(XCL_BO_SYNC_BO_FROM_DEVICE, size, offset);
+        return XMA_SUCCESS;
     }
-
-    return XMA_SUCCESS;
+    catch (const xrt_core::system_error& ex) {
```

Comment:
don't name the argument if it is not used, so in this case just do
`catch (const xrt_core::system_error&) {`

Reply:
comments addressed